.github/workflows/nodejs.yml

```diff
@@ -19,3 +19,6 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/actions/ci-setup
       - uses: ./.github/actions/ci-checks
+        env:
+          TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
```

Comment:
I suspect that this might have a problem running on PRs as secrets shouldn't be exposed to the actions triggered by such event (it would be a security concern)

Reply:
Ah, grim - true.

cc @jaredpalmer is there a solution here?